src/App.tsx

```diff
@@ -17,39 +80,23 @@ export const App: React.FC = () => {
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            onChange={handleQueryChange}
+            value={query}
           />
+          {filteredPeople.length === 0 && query
+          && <p>No matching suggestions</p>}
+
         </div>
 
         <div className="dropdown-menu" role="menu">
-          <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Bernard Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Antone Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Petronella de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Hercke</p>
+          {filteredPeople.length === 0 && query ? null : (
+            <div className={`dropdown-content ${isVisible ? 'visible' : ''}`}>
+              <PeopleList
+                posts={initialPerson}
```

Comment:
Why do you use `initialPerson` here?
I think, you should use filteredPeople to show which people have been founded after user search

Reply:
isn't fixed